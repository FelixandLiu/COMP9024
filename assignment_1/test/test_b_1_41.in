a
27226186
128.7024
a
62662281
242.7387
a
90222831
129.3002
a
96537850
224.6492
a
174639028
88078089
26.6867
a
24930075
587.6358
216.8322
a
10071781
178.9736
a
652854105
72806549
123.6198
a
27938247
116.3034
a
28751154
191.4413
a
44945263
40.5681
a
57078339
151.3078
a
54173992
176.0479
a
80912178
197.1808
a
98814593
13.4485
a
23151739
-719.0291
225.3938
a
45341562
28.2294
a
56753331
180.7695
a
20062073
216.8268
a
99101844
114.6084
a
11125989
208.5188
a
66026938
63.1983
a
98189893
235.2275
a
60162976
211.5542
a
88100190
229.2593
a
49637435
95.1512
a
52863926
162.976
a
53186614
36.0502
a
82781130
76.5025
a
79961413
13.0231
a
8762296
96153951
174.0192
a
65428426
25.212
a
62672738
101.9242
a
30056645
219.0569
a
10617616
-412.6455
66.0844
a
684550449
39623511
87.8964
a
12321030
37.1124
a
76888576
12.871
a
98245946
58.4531
a
62144098
129.5802
a
4237917
48865980
109.2038
p
g
q
