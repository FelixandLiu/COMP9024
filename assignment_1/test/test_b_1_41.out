Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Not valid. Enter a valid value: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Not valid. Enter a valid value: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Not valid. Enter a valid value: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Not valid. Enter a valid value: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Not valid. Enter a valid value: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Not valid. Enter a valid value: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Not valid. Enter a valid value: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Not valid. Enter a valid value: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> -----------------
Card ID: 10071781
Balance: $178.97
-----------------
-----------------
Card ID: 10617616
Balance: $66.08
-----------------
-----------------
Card ID: 11125989
Balance: $208.52
-----------------
-----------------
Card ID: 12321030
Balance: $37.11
-----------------
-----------------
Card ID: 20062073
Balance: $216.83
-----------------
-----------------
Card ID: 23151739
Balance: $225.39
-----------------
-----------------
Card ID: 24930075
Balance: $216.83
-----------------
-----------------
Card ID: 27226186
Balance: $128.70
-----------------
-----------------
Card ID: 27938247
Balance: $116.30
-----------------
-----------------
Card ID: 28751154
Balance: $191.44
-----------------
-----------------
Card ID: 30056645
Balance: $219.06
-----------------
-----------------
Card ID: 39623511
Balance: $87.90
-----------------
-----------------
Card ID: 44945263
Balance: $40.57
-----------------
-----------------
Card ID: 45341562
Balance: $28.23
-----------------
-----------------
Card ID: 48865980
Balance: $109.20
-----------------
-----------------
Card ID: 49637435
Balance: $95.15
-----------------
-----------------
Card ID: 52863926
Balance: $162.98
-----------------
-----------------
Card ID: 53186614
Balance: $36.05
-----------------
-----------------
Card ID: 54173992
Balance: $176.05
-----------------
-----------------
Card ID: 56753331
Balance: $180.77
-----------------
-----------------
Card ID: 57078339
Balance: $151.31
-----------------
-----------------
Card ID: 60162976
Balance: $211.55
-----------------
-----------------
Card ID: 62144098
Balance: $129.58
-----------------
-----------------
Card ID: 62662281
Balance: $242.74
-----------------
-----------------
Card ID: 62672738
Balance: $101.92
-----------------
-----------------
Card ID: 65428426
Balance: $25.21
-----------------
-----------------
Card ID: 66026938
Balance: $63.20
-----------------
-----------------
Card ID: 72806549
Balance: $123.62
-----------------
-----------------
Card ID: 76888576
Balance: $12.87
-----------------
-----------------
Card ID: 79961413
Balance: $13.02
-----------------
-----------------
Card ID: 80912178
Balance: $197.18
-----------------
-----------------
Card ID: 82781130
Balance: $76.50
-----------------
-----------------
Card ID: 88078089
Balance: $26.69
-----------------
-----------------
Card ID: 88100190
Balance: $229.26
-----------------
-----------------
Card ID: 90222831
Balance: $129.30
-----------------
-----------------
Card ID: 96153951
Balance: $174.02
-----------------
-----------------
Card ID: 96537850
Balance: $224.65
-----------------
-----------------
Card ID: 98189893
Balance: $235.23
-----------------
-----------------
Card ID: 98245946
Balance: $58.45
-----------------
-----------------
Card ID: 98814593
Balance: $13.45
-----------------
-----------------
Card ID: 99101844
Balance: $114.61
-----------------
Enter command (a,g,p,q,r, h for Help)> Number of cards on file: 41
Average balance: $128.70
Enter command (a,g,p,q,r, h for Help)> Bye.
